models/lung_nodule_ct_detection/configs/metadata.json

```diff
@@ -0,0 +1,72 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.1.0",
+    "changelog": {
+        "0.1.0": "complete the model package"
+    },
+    "monai_version": "0.9.0",
```

Comment:
Hi @Can-Zhao ,

Could you please help update to test and use the MONAI 0.9.1?
We would like to support the latest version in this important bundle.

Thanks in advance.